src/App.js

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
 import { BrowserRouter as Router, Route } from 'react-router-dom';
 
 import { GamePage, JoinPage } from './components';
 
+import './App.css';
```

Comment:
isn't this App.css imported at `index.js` or something? does the style apply without this line?